src/jsonpath.js

```diff
@@ -143,7 +162,8 @@ function JSONPath (opts, expr, obj, callback, otherTypeCallback) {
     this.flatten = opts.flatten || false;
     this.wrap = hasOwnProp.call(opts, 'wrap') ? opts.wrap : true;
     this.sandbox = opts.sandbox || {};
-    this.preventEval = opts.preventEval || false;
+    this.eval = opts.eval === undefined ? 'safe' : opts.eval;
```

Comment:
Especially given the importance of this option, I think we should throw if the argument is not one of the expected types (e.g., if there was a misspelling).

Reply:
Added the validation in
https://github.com/JSONPath-Plus/JSONPath/blob/d34dcce307f9116ca680845e1bcbf1433d51de66/src/jsonpath.js#L673-L675

Also a test
https://github.com/JSONPath-Plus/JSONPath/blob/d34dcce307f9116ca680845e1bcbf1433d51de66/test/test.eval.js#L29-L36